### PR TITLE
Add more information about where to find the Redirected values

### PR DESCRIPTION
**Pull request by d-ronnqvist**
Bug/issue #, if applicable: 

## Summary

This updated the `@Redirected` documentation to describe how and where the developer can find the redirect values for each page.

## Dependencies

None

## Testing

- Run `swift run docc preview Sources/docc/DocCDocumentation.docc`
- Navigate to http://localhost:8080/documentation/docc/redirected 

## Checklist

Make sure you check off the following items. If they cannot be completed, provide a reason.

- ~[ ] Added tests~
- [x] Ran the `./bin/test` script and it succeeded
- [x] Updated documentation if necessary

### CODE REVIEW

File: Sources/SwiftDocC/Semantics/Redirect.swift [hunk 1]
```diff
@@ -12,16 +12,19 @@ import Foundation
...
 
 
-/// A directive that specifies an additional URL for the page where the directive appears.
+/// A directive that specifies a previous URL for the page where the directive appears.
 ///
-/// Use this directive to declare a URL where a piece of content was previously located.
-/// For example, if you host the compiled documentation on a web server,
-/// that server can read this data and set an HTTP "301 Moved Permanently" redirect from
-/// the declared URL to the page's current URL and avoid breaking any existing links to the content.
+/// If the page has moved more than once you can add multiple  `Redirected` directives, each specifying one previous URL.
```
Comment (patshaughnessy) [inline Sources/SwiftDocC/Semantics/Redirect.swift:17]:
```suggestion
/// If the page has moved more than once you can add multiple  `Redirected` directives, each specifying one previous URL. For example:
///
/// ```
/// @Redirected(from: "old/path/to/this/page")
/// @Redirected(from: "another/old/path/to/this/page")
/// ```
```

File: Sources/SwiftDocC/Semantics/Redirect.swift [hunk 2]
```diff
@@ -12,16 +12,19 @@ import Foundation
...
-/// previous versions, @Redirected must be used as a top level directive.
+/// > Note: Starting with version 5.11, the `Redirected` directive is supported both top-level and as a member of a ``Metadata`` directive. In
+/// earlier versions, the `Redirected` directive is only supported as a top-level directive.
 ///
+/// ### Setting up Redirects
+///
+/// If you host your documentation on a web server, you can set a HTTP "301 Moved Permanently" redirect for each `Redirected` value to avoid breaking existing links to your content.
+///
+/// To find each page's `Redirected` values, pass the `--emit-digest` flag to DocC and read the "linkable-entities.json" file that this makes DocC write to the output directory. 
+/// Each "entity" in that file has a "path"---which represents the current relative path of that page---and an optional list of "redirects"---which represent all the `Redirected` values for page as they were authored in the markup. You can write either relative redirect values or absolute redirect values depending on what information you need to set HTTP "301 Moved Permanently" redirects on your web server.
```
Comment (patshaughnessy) [inline Sources/SwiftDocC/Semantics/Redirect.swift:27]:
```suggestion
/// To find each page’s Redirected values, pass the `--emit-digest` flag to DocC.
/// This flag instructs DocC to emit several additional JSON files containing
/// information (the “digest”) about the documentation build. One of these digest
/// files, `linkable-entities.json`, contains an array of “entities" (pages) of
/// documentation. Each of these has a path---which represents the current relative
/// path of that page---and an optional list of redirects---which represent all the
/// Redirected values for page as they were authored in the markup. You can write
/// either relative redirect values or absolute redirect values depending on what
/// information you need to use when setting HTTP “301 Moved Permanently” redirects
/// on your web server.
```

**Review by patshaughnessy — APPROVED** [review]
This is great; thank you! I left a couple of suggestions with a bit of word-crafting if you're interested.

**Comment by d-ronnqvist** [comment]
@swift-ci please test

File: Sources/SwiftDocC/Semantics/Redirect.swift (hunk 1 above)
Comment (d-ronnqvist) [reply Sources/SwiftDocC/Semantics/Redirect.swift]: Thanks. I like the examples but I wanted the code block to use syntax highlighting so I applied the changes manually in https://github.com/apple/swift-docc/pull/811/commits/a87ad1dd3379179c437b7be1a048f39c5259e6ef

File: Sources/SwiftDocC/Semantics/Redirect.swift (hunk 2 above)
Comment (d-ronnqvist) [reply Sources/SwiftDocC/Semantics/Redirect.swift]: I update this paragraph with some changes from this suggestion in https://github.com/apple/swift-docc/pull/811/commits/a87ad1dd3379179c437b7be1a048f39c5259e6ef

**Pull request merged**